test/parallel/test-http-localaddress.js

```diff
@@ -19,6 +19,7 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+// Flags: --expose-internals
```

Comment:
Why is this now needed? The same in the part below.

Reply:
The reason for this is that there is a call of the function `common.hasMultiLocalhost()` [below](https://github.com/nodejs/node/pull/22432/files/6a28092d8dc31fcc14e74b091c237f548dd3112e#diff-1dfc1917bb065339bf24fa88636a2486R25) this line which now uses [require('internal/test/binding')](https://github.com/nodejs/node/pull/22432/files/6a28092d8dc31fcc14e74b091c237f548dd3112e#diff-9ae71bf1b8bff0e90c13e5b82ee79f09R465).